modules/testsuite/cxf-tests/pom.xml

```diff
@@ -498,6 +498,272 @@
+                                      </property>
+                                  </properties>
+                              </configuration>
+                          </execution>
+                          <execution>
+                              <id>testsuite-ssl-mutual-auth</id>
+                              <phase>pre-integration-test</phase>
+                              <goals>
+                                  <goal>execute</goal>
+                              </goals>
+                              <configuration>
+                                  <scripts>
+                                      <script>file:///${basedir}/src/test/scripts/jbws-testsuite-ssl-mutual-auth-elytron.groovy</script>
+                                  </scripts>
+                                  <properties>
+                                      <property>
+                                          <name>inputFile</name>
+                                          <value>${jboss.home}/standalone/configuration/standalone.xml</value>
+                                      </property>
+                                      <property>
+                                          <name>outputFile</name>
+                                          <value>${jboss.home}/standalone/configuration/jbws-testsuite-ssl-mutual-auth.xml</value>
+                                      </property>
+                                      <property>
+                                          <name>serverLog</name>
+                                          <value>jbws-testsuite-ssl-mutual-auth.log</value>
+                                      </property>
+                                      <property>
+                                          <name>keystorePath</name>
+                                          <value>${project.build.directory}/test-classes/test.keystore</value>
+                                      </property>
+                                      <property>
+                                          <name>truststorePath</name>
+                                          <value>${project.build.directory}/test-classes/test.truststore</value>
+                                      </property>
+                                  </properties>
+                              </configuration>
+                          </execution>
+                          <execution>
+                              <id>testsuite-default-config-tests</id>
+                              <phase>pre-integration-test</phase>
+                              <goals>
+                                  <goal>execute</goal>
+                              </goals>
+                              <configuration>
+                                  <scripts>
+                                      <script>file:///${basedir}/src/test/scripts/jbws-testsuite-default-config-tests-elytron.groovy</script>
```

Comment:
There were no changes needed to the server configuration related to picketbox
for this category of tests.  I chose to maintain the reference to the original 
file and not make a duplicate with just a name change.

Reply:
@rsearls  This script is to create elytron configuration and enable the elytron in undertow and ejb to test.  We are going to test with picketbox, we have to configure it for test and it can't coexist in undertow and ejb subsystems.